tests/rptest/tests/delete_records_test.py

```diff
@@ -237,13 +258,17 @@ def test_delete_records_topic_start_delta(self, cloud_storage_enabled):
                                      truncate_offset_start + 5):
             # Perform truncation
             low_watermark = self.delete_records(topic, 0, truncate_offset)
-            assert low_watermark == truncate_offset, f"Expected low watermark: {truncate_offset} observed: {low_watermark}"
+            assert (
+                low_watermark == truncate_offset
+            ), f"Expected low watermark: {truncate_offset} observed: {low_watermark}"
 
             # Assert correctness of start and end offsets in topic metadata
             topic_info = self.get_topic_info(topic)
             assert topic_info.id == 0, f"Partition id: {topic_info.id}"
-            assert topic_info.start_offset == truncate_offset, f"Start offset: {topic_info.start_offset}"
-            assert topic_info.high_watermark == num_records, f"High watermark: {topic_info.high_watermark}"
+            assert (topic_info.start_offset == truncate_offset
```

Comment:
confused where the meat and potatoes is in this commit

Reply:
Ditto. These changes got in by accident.